README.md

```diff
@@ -10,10 +8,37 @@
 6. Redirect_to from a controller action to a show page
 7. Use a named route for the new page and put it on the index page
 
-## Notes
 
-build a new/create action for a new student
+## Instructions
+
+There are a some additional RSpec specs for this lesson, including our first use of controller tests. You can find the three new tests that have to pass here:
+
+* **Controller Specs** - `specs/controllers/students_controller_spec.rb`
+
+* **Capybara Specs** - `specs/features/student_spec.rb:37`
+
+For the use case scenario, after this lab is completed your application should have the following behavior:
+
+1. User fills out the form on `students/new`
+
+2. When the form is submitted a new record is created in the database
+
+3. After the user fills out the form they are redirected to the `show` page that renders the `student` record that they created
+
+
+## Keys to remember
+
+* Your `create` method should create a new `Student` instance and store it in an instance variable
```

Comment:
Go ahead and drop these keys to remember. Maybe just the line about make sure to use route helpers